.github/workflows/main.yml

```
@@ -60,11 +60,12 @@ jobs:
           xvfb-run --auto-servernum npm run unit-with-coverage
           xvfb-run --auto-servernum npm run assert-unit-coverage
 
-      - name: Run unit tests on Firefox
-        env:
-          FIREFOX: true
-        run: |
-          xvfb-run --auto-servernum npm run funit
+      # Temporarily disabled due to FF Nightly flakes.
```

Fun fact: the official shorthand is not “FF” but rather “Fx”. https://mths.be/fx

```suggestion
      # Temporarily disabled due to Firefox Nightly flakes.
```